config/database.yml

```diff
@@ -20,4 +20,4 @@ test:
 
 production:
   <<: *default
-  database: library_mk4_prod
+  database: library_mk4_dev
```

Comment:
Did we combine dev/prod databases so there is only one? This feels like a bad idea.

Reply:
We've been running in dev mode exclusively. Switching to Prod mode now for better performance. I'll copy everything from the dev DB to prod and change this back to prod DB! Thanks!